Documentation/compel.txt

```diff
@@ -97,7 +97,7 @@ Following steps are performed to infect the victim process:
     - execute system call: *int compel_syscall(ctl, int syscall_nr, long *ret, int arg ...);*
     - infect victim: *int compel_infect(ctl, nr_thread, size_of_args_area);*
     - cure the victim: *int compel_cure(ctl);* //ctl pointer is freed by this call
-    - Resume victim: *int compel_resume_task(pid, orig_state, state);*
+    - Resume victim: *int compel_resume_task(pid, orig_state, state, stop_signo);*
```

Comment:
May be it's worth not to change signature of public function `compel_resume_task` but just add a separate function like `compel_resume_task_with_sig` or something like that.

Reply:
Thanks, that's a great point, fixed!

Reply:
Great, but I think it makes sense to add info about a new `compel_resume_task_sig` function into compel.txt

Reply:
Added some info about compel_resume_task_sig and compel_parse_stop_signo. But I am not sure about the correct writing of this info on the man page.